.github/workflows/ai-runner-models.yaml

```diff
@@ -80,6 +80,9 @@ jobs:
       matrix:
         gpu: [rtx-4090]
     runs-on: ${{ matrix.gpu }}
+    concurrency:
+      group: download-build-tensorrt-models
```

Comment:
should the group also reference `matrix.gpu` var; so that if later we have more gpus, each can have a separate build running? not sure how job specific concurrency groups work.